### PR TITLE
SPACE FISHING

### PR DESCRIPTION
**Pull request by Iamgoofball**
read title

give the miners fishing poles

add a way for them to get bait, probably the ore redemption machine

will write an OP someday(tm)

### CODE REVIEW

**Comment by Iamgoofball** [comment]
ps

i also need sprites

if you value your sanity don't look at the DMI

except the fishing pole that one is good

**Comment by MrPerson** [comment]
Fishing adds nothing interesting to mining. Mining can already get minerals. Cargo can already get food. Science can already get pico manipulators (that are somehow inside a fish?!?!?). The chef can already make loads of bread. This just devalues all those jobs with a boring, lame gimmick.

Let me restate that: This shit is boring. Really boring. Why not hook into the existing mining mobs? Shoot, there's loads of interesting stuff you can bullshit up with those. Instead of having magic "bread fish" or "tobacco fish".

**Comment by ShadowLordAlpha** [comment]
Why not one fish that has a few different skins and can be sent to the cook to make plates of fish sticks with

**Comment by Aranclanos** [comment]
this is awful

File: code/modules/mining/fishing/fish_types.dm [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	desc = "A fish."
+	harvest_drops = list(new /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/deus(), // Support the botanists by letting them skip the mutation phase to get some good plants
+						new /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/deus(),
+						new /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/deus(),
+						new /obj/item/weapon/reagent_containers/food/snacks/grown/tobacco/space(),
+						new /obj/item/weapon/reagent_containers/food/snacks/grown/tobacco/space())
+	harvest_times = 3
+
+/obj/item/weapon/fish/bread
+	name = "loafer"
```
Comment (Incoming5643) [inline code/modules/mining/fishing/fish_types.dm]: https://www.youtube.com/watch?v=0RpdPzJgaBw

Comment (Iamgoofball) [inline code/modules/mining/fishing/fish_types.dm]: literally had no idea this was a thing

**Comment by bgobandit** [comment]
"Adding more bananium into the round is a serious issue that needs to be discussed."

Fishing is a drop in the bucket compared to [REDACTED PLS NO NERFS]

**Comment by optimumtact** [comment]
Who cares if it's boring, interesting little side things like this are what makes ss13 great

**Comment by Thunder12345** [comment]
@Aranclanos 
You're just mad you couldn't finish your own space fishing

**Comment by Miauw62** [comment]
MAY MAYS
Op 12-jun.-2015 10:30 schreef "Thunder12345" <notifications@github.com>:

**Comment by bgobandit** [comment]
can we at least get separate descriptions for the different fish, fuck I'll even write them

**Comment by bgobandit** [comment]
moar notes:

1) no carp/megacarp?
2) also, just for kicks (and to clog up the RNG so it does not produce appreciably gamebreaking amounts of stuff) there should be a chance to fish up junk. shoes, empty cans, body parts, all those standard fishing junk items.
3) if you want to go full minecraft you can tie the chance of getting junk to how upgraded your fishing pole is, which will also stem the OH GOD DIAMOND URANIUM BANANIUM flood

**Comment by Chocobro** [comment]
oh my god these sprites goof
they're just
they're just CIRCLES

for bait you could sprite it as bait boxes that contain the actual bait (often maggots in real fishing, otherwise a compound)
fish_iron? fish_gold, fish_silver? pretty obvious that you're going to make it like a secondary way of getting minerals. but what are the scales used for? i can do these sprites if you like. but don't ping me on IRC when it's 3am. then i won't do them. i'll be asleep.

**Comment by Iamgoofball** [comment]
@Chocobro if you notice, the scales are ore subtypes

you can stuff them in the redemption machine/smelter and get ores

**Comment by Nienhaus** [comment]
Those sprites are 10/10 :+1:

**Comment by Durkel** [comment]
fishing lvls?

**Comment by Iamgoofball** [comment]
raw_lobster_detail

**Comment by Chocobro** [comment]
35
free fires just ask

**Comment by ike709** [comment]
#5000

File: code/modules/food&drinks/kitchen machinery/processor.dm [hunk 2]
```diff
@@ -34,6 +34,15 @@
 	var/output
 	var/time = 40
 /datum/food_processor_process/proc/process_food(loc, what, var/obj/machinery/processor/processor)
+	if(istype(what, /obj/item/weapon/fish))
```
Comment (KazeEspada) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: Is this snowflake code?

Comment (KazeEspada) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: Alright, see this line? See it? Are you sure that ever fucking time I process a piece of corn, that I should make the food processor make sure its not a fucking fish?

Comment (KazeEspada) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: This is terrible snowflake code.

Comment (MrPerson) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: much oo

**Comment by Iamgoofball** [comment]
account banned from scaperune

rip

**Comment by optimumtact** [comment]
sad

**Comment by TechnoAlchemisto** [comment]
pour one out

**Pull request closed without merging**